### DISCUSSION TITLE
How to reuse the ModelView data table?

### DISCUSSION THREAD

**Discussion by irux** [jowilf/starlette-admin#492] [Q&A]
Hello, I would like to reuse the data table that comes with the ModelView list.html template. I would like to add more logic to this like syncing some numbers on the screen with the checked objects of the table. How would be the best way to do it ? Any recommendations? it seems a little bit difficult to extend and customize for that propose.

**Comment by hasansezertasan** [comment]
I couldn't understand what you meant but you can easily override `list.html`

**Reply by irux** [reply]
@hasansezertasan right now starlette-admin uses the list.html file for the modelviews https://github.com/jowilf/starlette-admin/blob/main/starlette_admin/templates/list.html. I want to extend/override the list.html template with jinja and be able to have access to the table. I see that starlette-admin uses a file for the implementation of this js logic of the table https://github.com/jowilf/starlette-admin/blob/main/starlette_admin/statics/js/list.js The problem is I don't see a way to use the instance of the table there because it is encapsulated. Any ideas how to give more functionalities to the table/track changes on the changes?

**Reply by hasansezertasan** [reply]
I understand clearly now but I can not think of any solution for that.

Sorry about that 😞, even if DataTableJS is a huge gain, it sometimes locks the (non-JS) devs.
